### PR TITLE
feat(init): make magefile example include standard go library tools

### PR DESCRIPTION
**Pull request by viktorvoltaire**

### CODE REVIEW

**Review by odsod — APPROVED** [review]
:100:

File: example/.mage/magefile.go [hunk 1]
```diff
@@ -24,20 +36,16 @@ func init() {
...
-Mage-tools has been successfully initialized!
-
-To get started, have a look at the magefile.go in the .mage directory,
-and look at https://github.com/einride/mage-tools#readme to learn more
-`)
-}
-
 func All() {
 	mg.Deps(
 		mg.F(mgconvco.ConvcoCheck, "origin/master..HEAD"),
```
Comment (odsod) [inline example/.mage/magefile.go:41]: should this be `master` or `main`...? :joy:

Comment (viktorvoltaire) [reply example/.mage/magefile.go:41]: > should this be `master` or `main`...? joy

:rofl: could this be figured out from command line? Rather then hardcoding it :P

Comment (Edholm) [reply example/.mage/magefile.go:41]: `git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@'` seems to work

Comment (odsod) [reply example/.mage/magefile.go:41]: @Edholm maybe worth adding as a mage-tools helper function?

Comment (Edholm) [reply example/.mage/magefile.go:41]: Yep thought so too. I'll see if I have time to do that today.

Created #83 for now.

Comment (viktorvoltaire) [reply example/.mage/magefile.go:41]: 👌

**Pull request merged**